internal/proxy/clientgo_cached_proxy_resolver.go

```diff
@@ -34,12 +34,13 @@ func NewCacheBasedProxy(ctx context.Context,
 	kongUpdater KongUpdater,
 	diagnostic util.ConfigDumpDiagnostic,
 	proxyRequestTimeout time.Duration,
+	kongWorkspace string,
```

Comment:
question: it looks like we added this string to the proxy but the attribute is not being used anywhere... am I missing something? Where is this being used?

Reply:
good catch. actually, this set is not needed anymore since we already use configuration globally.